epoch1/cuda/ee_mumu/SubProcesses/Makefile

```diff
@@ -1,23 +1,27 @@
-CUARCHNUM=70
-#CUARCHNUM=61 #(For Pascal Architecture Cards)
-LIBDIR=../../lib
+LIBDIR =../../lib
 INCDIRS=-I. -I../../src -I../../../../../tools/
 MODELLIB=model_sm
-CXXFLAGS= -O3 $(INCDIRS) -DUSE_NVTX -Wall -Wshadow ${MGONGPU_CONFIG}
-###CXXFLAGS= -O2 $(INCDIRS) -DUSE_NVTX -Wall -Wshadow
-CUARCHFLAGS= -arch=compute_$(CUARCHNUM)
-# CUARCHFLAGS= -gencode arch=compute_$(CUARCHNUM),code=sm_$(CUARCHNUM)
-CUFLAGS= -O3 $(INCDIRS) -DUSE_NVTX $(CUARCHFLAGS) -use_fast_math -lineinfo ${MGONGPU_CONFIG}
-# Without -maxrregcount: baseline throughput: 6.5E8 (16384 32 12) up to 7.3E8 (65536 128 12)
-###CUFLAGS+= --maxrregcount 160 # improves throughput: 6.9E8 (16384 32 12) up to 7.7E8 (65536 128 12)
-###CUFLAGS+= --maxrregcount 128 # improves throughput: 7.3E8 (16384 32 12) up to 7.6E8 (65536 128 12)
-###CUFLAGS+= --maxrregcount 96 # degrades throughput: 4.1E8 (16384 32 12) up to 4.5E8 (65536 128 12)
-###CUFLAGS+= --maxrregcount 64 # degrades throughput: 1.7E8 (16384 32 12) flat at 1.7E8 (65536 128 12)
-CUINC=/usr/local/cuda/targets/x86_64-linux/include
+CXXFLAGS= -O3 -std=c++11 $(INCDIRS) $(USE_NVTX) -Wall -Wshadow ${MGONGPU_CONFIG}
 LIBFLAGS= -L$(LIBDIR) -l$(MODELLIB)
-CULIBFLAGS= -L/usr/local/cuda/targets/x86_64-linux/lib
-NVCC=nvcc
 CXX=g++
+NVCC?=$(shell which nvcc)
+ifeq (, $(NVCC))
+	NVCC=@echo No nvcc to run nvcc
+else
+	USE_NVTX?=-DUSE_NVTX
+	CUARCHNUM=70
+	#CUARCHNUM=61 #(For Pascal Architecture Cards)
```

Comment:
Actually, I do have a minor issue here.
My emacs gives me a warnings about indented comments.
As I got used to avoiding indeneted comments, I thought that make would fail, actually it does not.
But I just understood that there is a reason emacs complains, indented comments are parsed and executed: https://stackoverflow.com/a/18363477

Anyway, I suggest that you fix this in your next PR #63. I will repeat the comment there.